lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
```

Comment:
If nil is passed, the program crashes on dice.inject (NoMethodError) - this line is intended to result in more predictable behavior.